frontend/src/entities/group/detail/group-detail.tsx

```diff
@@ -0,0 +1,103 @@
+import { getGroup, getGroupUsers } from '../group.reducer';
+import { Header, SpaceBetween } from '@cloudscape-design/components';
+import DetailsContainer from '../../../modules/details-container';
+import { IGroup } from '../../../shared/model/group.model';
+import ContentLayout from '../../../shared/layout/content-layout';
+import GroupDetailActions from './group-detail.actions';
+import Table from '../../../modules/table';
+import { groupUserColumns, visibleGroupUserColumns } from '../../user/user.columns';
+import { IGroupUser } from '../../../shared/model/groupUser.model';
+import NotificationService from '../../../shared/layout/notification/notification.service';
+
+export function GroupDetail () {
+    const dispatch = useAppDispatch();
+    const navigate = useNavigate();
+    const {groupName} = useParams();
+    const [group, setGroup] = useState<IGroup>(null);
+    const [groupUsers, setGroupUsers] = useState<IGroupUser>([]);
+    const [initialLoaded, setInitialLoaded] = useState(false);
+    const [groupUsersLoaded, setGroupUsersLoaded] = useState(true);
+    const notificationService = useMemo(() => NotificationService(dispatch), [dispatch]);
+
+    const groupDetails = new Map<string, ReactNode>();
+    groupDetails.set('Name', group?.name);
+    groupDetails.set('Description', group?.description);
+
+    useEffect(() => {
+        if (initialLoaded === false) {
+            dispatch(getGroup(groupName)).then((response) => {
+                if (response.payload) {
+                    setGroup(response.payload.data.group);
+                    setInitialLoaded(true);
+                } else {
+                    navigate('/404');
+                }
+            });
+        }
+    }, [groupName, initialLoaded, dispatch, navigate]);
+
+    useEffect(() => {
+        if (initialLoaded === true) {
+            dispatch(getGroupUsers(groupName)).then((response) => {
+                if (response.payload) {
+                    setGroupUsers(response.payload.data);
+                } else {
+                    notificationService.generateNotification(
+                        `Failed to fetch ${groupName} users.`,
+                        'error'
+                    );
+                }
+                setGroupUsersLoaded(true);
+            });
+        }
+    }, [initialLoaded, groupName, dispatch, notificationService]);
+
+    return (
+        <ContentLayout header={<Header variant='h1'>{groupName}</Header>}>
+            <SpaceBetween direction='vertical' size='xxl'>
+                <DetailsContainer
+                    columns={1}
+                    header='Group details'
+                    actions={GroupDetailActions()}
+                    info={groupDetails}
+                    loading={!initialLoaded}
+                />
+                {/*<Container>*/}
+                <Table
+                    tableName='Group member'
+                    // tableType={tableType}
```

Comment:
Should these just be removed? Same with the commented out Container on line 84

Reply:
And line 97

Reply:
I left them because they will be needed when I add user management in the next task. Since this isn't going into develop I figured it would be okay.

Reply:
ah okay